examples/stores/opentelemetry_example.py

```diff
@@ -0,0 +1,27 @@
+from observers.observers import wrap_openai
+from observers.stores.opentelemetry import OpenTelemetryStore
+from openai import OpenAI
+
+
+# Use your usual environment variables to configure OpenTelemetry
+# Here's an example for Honeycomb
+# export OTEL_SERVICE_NAME=<identifiable-service-name>
+# export OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+# export OTEL_EXPORTER_OTLP_ENDPOINT="https://api.honeycomb.io"
+# export OTEL_EXPORTER_OTLP_HEADERS="x-honeycomb-team=<ingest-key>"
+
+store = OpenTelemetryStore()
+
+openai_client = OpenAI(base_url="http://localhost:11434/v1", api_key="unused")
```

Comment:
```suggestion
openai_client = OpenAI()
```

Let's use normal OpenAI client instead of Ollama running locally since most people will have their OpenAI key set and it's one less dependency needed to run the example :)